incubator/hnc/pkg/stats/metrics.go

```diff
@@ -0,0 +1,44 @@
+package stats
+
+import (
+	"context"
+
+	ocstats "go.opencensus.io/stats"
+	ocview "go.opencensus.io/stats/view"
+	ctrl "sigs.k8s.io/controller-runtime"
+)
+
+var (
+	hierConfigReconcileTotal = ocstats.Int64("hierconfig_reconcile_total", "The number of HierConfig reconciliations happened", "times")
+	hierConfigWritesTotal    = ocstats.Int64("hierconfig_writes_total", "The number of HierConfig writes happened during HierConfig reconciliations", "times")
+)
+
+var (
+	hierReconcileView = &ocview.View{
+		Name:        "hnc/hierconfig_reconciler/reconcile_total",
```

Comment:
Nit suggestion: about about `hnc/reconcilers/hierconfig/total`? Why does "reconcile" need to be repeated?

The alternative to `reconcilers` will eventually be things like `validators` and (maybe) `mutators`, while the alternatives to `hierconfig` will be `objects` and `hncconfig`.

Reply:
Sg, done. The other one is renamed as `hnc/reconcilers/hierconfig/hierconfig_writes_total` since there is also a `hnc/reconcilers/hierconfig/namespace_writes_total` in addition.

Reply:
sgtm